apps/portal/js/wizard.js

```diff
@@ -426,10 +440,22 @@ function getChartConfig(providerConfig) {
         async: false,
         success: function (chartConfig) {
             if (!chartConfig.error) {
-                registerAdvancedChartUI(chartConfig);
                 var chartHbs = Handlebars.compile($('#ui-config-hbs').html());
-                $("#chart-config").html(chartHbs(chartConfig));
-                $("#preview").removeAttr("style");
+                if (Object.keys(chartConfig[0]).indexOf(groups) > -1) {
+                    configGroup = (chartConfig[0].grouping);
+                    for (i = 0; i < configGroup.length; i++) {
```

Comment:
make sure to declare i, eg: var i . Otherwise this can leak into the global scope.